### ISSUE TITLE
[Feature Request] Manage custom emoji sets

### ISSUE THREAD

**Issue by rafitamolin** [telegramdesktop/tdesktop#25304]
### Is your feature request related to a problem?

On other official clients such as Telegram for Android, under Settings > Stickers and Emoji > Emoji, you can rearrange the order of custom emoji sets.

### Describe the solution you'd like

Create a "Manage custom emoji sets" section in Telegram Desktop in Settings > Chat Settings similar to the existent "Manage sticker sets".

### Describe alternatives you've considered

.

**Comment by RememberTheAir** [comment]
I wouldn't label it as a "feature request" but it's rather a missing feature, which lack forces the user to pick up their mobile phone

**Comment by rafitamolin** [comment]
> I wouldn't label it as a "feature request" but it's rather a missing feature, which lack forces the user to pick up their mobile phone

Yeah but between the three available categories I thought "Feature Request" was the best one for it, as I don't consider it a bug and neither would the developer.

**Comment by RememberTheAir** [comment]
I usually just replace the leading "[feature request]" in the title with "[missing]" just to highlight that this is not a request but rather something that _needs_ to be implemented sooner or later
